### PR TITLE
applied prettifier formatting to entire repo

### PR DESCRIPTION
**Pull request by samgutentag**
a collection of pseudo well broken up commits with outputs from running `npx prettier --write --prose-wrap always _` on directories and files.

includes fixups for admonitions and other table formatting fixes

### CODE REVIEW

**Comment by samgutentag** [comment]
@kent-williams this is a large PR, and I wasn't sure who else to request a review to, happy to talk this one through

**Comment by jthiller** [comment]
I'm just gonna throw out the wild opinion that we shouldn't hard wrap text. It makes small changes more fragile if the IDE isn't set up for it.

**Comment by abhay** [comment]
> I'm just gonna throw out the wild opinion that we shouldn't hard wrap text. It makes small changes more fragile if the IDE isn't set up for it.

add an .editorconfig file and 🙏🏼 ?

File: sidebarsDocs.js [hunk 1]
```diff
@@ -72,1187 +71,1284 @@ module.exports = {
...
+        "api/blockchain/dc-burns",
+        "api/blockchain/challenges",
+        "api/blockchain/elections",
+        "api/blockchain/state-channels",
+        "api/blockchain/assert-locations",
+      ],
       collapsed: true,
     },
-    'api/console',
+    "api/console",
```
Comment (Perronef5) [inline sidebarsDocs.js:142]: I've seen in a couple of our repos we prefer `single-quote`. Should we stick to that convention?

Comment (samgutentag) [reply sidebarsDocs.js:142]: I'm happy to do so! default prettier settings use double, but with the inclusion of a `.prettierrc` that's an easy override to conform to other repos

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 80,
+  "proseWrap": "always",
+  "singleQuote": false,
```
Comment (Perronef5) [inline .prettierrc]:
```suggestion
  "singleQuote": true,
```

Comment (Perronef5) [inline .prettierrc]:
```suggestion
  "singleQuote": true,
```

**Review by Perronef5 — APPROVED** [review]
LGTM

**Pull request merged**